cli/lib/kontena/cli/master/join_command.rb

```diff
@@ -9,9 +9,9 @@ class JoinCommand < Kontena::Command
 
     def execute
       params = []
-      params << "--join #{self.invite_code.shellescape}"
+      params += ["--join", self.invite_code.shellescape]
```

Comment:
I doubt the shellescape helps here, because there's no shell there to parse them in `Kontena.run`? Only matters if the invite_code/name contains shell metacharacters, I suspect those will get mangled and broken ATM.

(could also be `--join=#{...}`)